config/deploy.rb

```diff
@@ -26,7 +23,7 @@
 set :pty, true
 
 # Default value for linked_dirs is []
-append :linked_files, 'config/database.yml', 'config/credentials/staging.key', 'config/environments/staging.rb'
+append :linked_files, 'config/database.yml', 'config/credentials/staging.key', 'config/environments/staging.rb', 'config/credentials/production.key'
```

Comment:
Should we include `config/environments/production.rb` as well? If we need it in the future?